### PR TITLE
wg-multitenancy annual report

### LINKED ISSUE #6390: 2021 Annual Report: WG Multitenancy
Chairs: @srampal @tashimi 
Liaison: @liggitt

Actions for the chair/organizer of the community group:
- [ ] Consult your community group to complete draft of report
- [ ] If needed, consult with Steering Committee liaison on any private issues or concerns that arrise while completing report
- [ ] Submit PR to git.k8s.io/community repo with copy of report, assigning your Steering Committee liaison for review
- [ ] Steering Committee reviews report, and may make comments/ask questions
- [ ] Once all comments are addressed, Steering Committee will approve report to merge
- [ ] If needed, any follow up items may be brought to the "Chairs, Tech Leads, and Organizers" meeting in April

Once all the above items are complete, this issue may be `/close`'d

Key dates:
- Initial PR to communtiy repo should be opened by February 14, 2022
- PR should be reviewed and merged by March 1st, 2022

More detailed information on the annual reports process is available [here](https://git.k8s.io/community/committee-steering/governance/annual-reports.md).

Your group's report template is available here: https://git.k8s.io/community/wg-multitenancy/annual-report-2021.md

If you have any questions or concerns about this process, you may reach the Steering Committee via the following methods:
- Slack/E-mail your liaison directly
- The public #steering-committee channel in Slack
- The public mailing list steering@kubernetes.io
- The private mailing list steering-private@kubernetes.io for any private or sensitive issues.

/wg multitenancy
/committee steering

### PR DESCRIPTION
**Pull request by tashimi**
**Which issue(s) this PR fixes**:

Fixes #6390

### CODE REVIEW

**Comment by tashimi** [comment]
@rjbez17 @Fei-Guo @JimBugwadia please take a look!

**Comment by rjbez17** [comment]
lgtm

**Comment by Fei-Guo** [comment]
lgtm

File: wg-multitenancy/annual-report-2021.md [hunk 1]
```diff
@@ -5,56 +5,57 @@
 1. What work did the WG do this year that should be highlighted?
    For example, artifacts, reports, white papers produced this year.
 
-   -
-   -
-   -
+   - Hierarchical Namespace Controller: work continued (reached 1.0 in April 2022), moved into its own k8s repo out of WG incubation in a major project milestone
+   -- https://github.com/kubernetes-sigs/hierarchical-namespaces 
+   - Vertical Cluster Project: moved out of WG incubation and into its own k8s repo in a major milestone
```
Comment (JimBugwadia) [inline wg-multitenancy/annual-report-2021.md:10]: Virtual Cluster Project

**Review by JimBugwadia — CHANGES_REQUESTED** [review]
minor correction, looks great overall!

**Comment by devdattakulkarni** [comment]
looks good. 👍

File: wg-multitenancy/annual-report-2021.md [hunk 2]
```diff
@@ -5,56 +5,57 @@
...
-- [ ] Updates provided to sponsoring SIGs in 2021
-      - [$sig-name](https://git.k8s.io/community/$sig-id/)
-        - links to email, meeting notes, slides, or recordings, etc
-      - [$sig-name](https://git.k8s.io/community/$sig-id/)
-        - links to email, meeting notes, slides, or recordings, etc
-      -
+- [ x] [README.md] reviewed for accuracy and updated if needed
+- [ x] WG leaders in [sigs.yaml] are accurate and active, and updated if needed
+- [ x] Meeting notes and recordings for 2021 are linked from [README.md] and updated/uploaded if needed
+- [ x] Updates provided to sponsoring SIGs in 2021
```
Comment (liggitt) [inline wg-multitenancy/annual-report-2021.md:57]: links to the mailing list threads or meetings or recordings where this happened would be great for folks interested in the summaries

File: sigs.yaml [hunk 3]
```diff
@@ -3111,22 +3111,22 @@ workinggroups:
   label: multitenancy
   leadership:
     chairs:
-    - github: srampal
-      name: Sanjeev Rampal
-      company: Cisco
-    - github: tashimi
+     - github: tashimi
```
Comment (liggitt) [inline sigs.yaml:3114]: verify script is unhappy with this line... is indenting right?

**Comment by liggitt** [comment]
thanks for the update, looks good. one nit from CI on the yaml file and one request for any links to the sig updates you have

File: sigs.yaml [hunk 4]
```diff
@@ -3111,22 +3111,22 @@ workinggroups:
...
       name: Tasha Drew
       company: VMware
     emeritus_leads:
     - github: davidopp
       name: David Oppenheimer
+    - github: srampal
```
Comment (liggitt) [inline sigs.yaml:3120]: oh, would be good to get an ack on this move from @srampal or a reference to where this was communicated

Comment (tashimi) [reply sigs.yaml:3120]: Communicated here: https://groups.google.com/g/kubernetes-wg-multitenancy/c/fX3ZnmKkDL0 

I asked @srampal to do this several months ago but he hasn't replied, he hasn't been part of the group for about two years now, so removing him and adding the new chairs is appropriate.

File: sigs.yaml (hunk 3 above)
Comment (JimBugwadia) [reply sigs.yaml]: done!

**Comment by liggitt** [comment]
verify error:
```
go run ./generator/app.go
Validating sigs.yaml
ERROR: wg-multitenancy: emeritus leads should not have company specified; company specified for: Sanjeev Rampal
exit status 1
```

**Comment by liggitt** [comment]
opened https://github.com/kubernetes/community/pull/6672 which would have made that error more apparent in the CI job log

**Pull request closed without merging**